src/main/java/StringManipulation.java

```diff
@@ -1,32 +1,99 @@
-public class StringManipulation implements StringManipulationInterface {
```

Comment:
You have two "public class String....implements StringManipulationInter...{ ."

Reply:
The - before the line indicates that it was removed from the last commit